server/Helpers/movie.js

```diff
@@ -1,5 +1,30 @@
+import Axios from "axios";
 import MovieCommentModel from "../Schemas/Movie";
 
+const getRecommendation = async (req, res) => {
```

Comment:
Helpers don't take `(req, res)` object. **Helpers** are function that can be used anywhere (independently of the req/res) to do something (like interact with the DB or get movies from an external api, for exemple).
**Controllers** should receive the raw request (with req/res) and call Helpers to achieve the routes goal.

- Rename this function to something more precise like: `getRandomMoviesFromPCT` (PCT === Pop corn Time)
- Return just a list of movies from it
- Create a `getRecommendation` function in Controllers (`server/Controllers/movie.js`)
- call the `getRandomMoviesFromPCT` helper from the `getRecommendation` Controller


Your Controller should look a bit like this:
```node
const getRecommendation = async (_req, res) => {
  try {
    const movieList = await getRandomMoviesFromPCT();
    res.send({ list: movieList });
  } catch (e) {
    console.error(e);
    res.sendStatus(500);
  }
};
```

Reply:
Pop corn type doesn't work ! Check before push if it's fixe, else change source !

Reply:
Must check before render if pop corn url is always down !